diagnostic.md

```diff
@@ -73,20 +85,38 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --request POST http://www.example.com/donuts
+--header "Content-type: application/json" --data '{ "name": "French Cruller",
+"price": "$0.99",
+"done": false,
+"list_id": 3 }'
 ```
 
 Write an AJAX request to create a single donut on the server using JSON.
 
 ```js
-let createDonut = /* your answer here */;
+let createDonut = function(event) {
+  $.ajax({
+    url: 'http://www.example.com/donuts/3',
+    type: 'POST',
+    data: donut
+  });
```

Comment:
Also, this URL is wrong - you don't know yet what ID number the back end will assign to the new object that you're creating. That's why you send things to `../donuts` (no ID) when you create things.